### PR TITLE
Feature/meetup page

### PR DESCRIPTION
**Pull request by samrosecruz**
Todo on a different branch: navigation menu

As for this branch, it's ready to go 👍 
meetup

### CODE REVIEW

File: web/static/css/components/meetup-speakers.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@import "../variables";
+
+.meetup-speakers {
+  & {
+    margin-bottom: $pad * 10;
+  }
+
+  &:nth-of-type(2) {
```
Comment (riacataquian) [inline web/static/css/components/meetup-speakers.scss]: TIL @samrosecruz

File: web/templates/meetup/index.html.slim [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    h1.headline
+      | Alchemist PH |> Meetup 
+      span.digitize
+        | 02
+  .meetup-location
+    .icon
+      i.ion-md-pin
+    .details
+      h4.venue AElogica HQ, Bonifacio Global City
+      p.time June 30, 2016, Thurday. 7PM to 9PM
```
Comment (riacataquian) [inline web/templates/meetup/index.html.slim]: `Thursday` @samrosecruz :)

File: web/templates/meetup/index.html.slim [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+.meetup-splash
+  .date-box
+    span.month Jun
+    span.date 30
+  header.site-type
+    h2.prelude
+      | Upcoming Meetup
+    h1.headline
+      | Alchemist PH |> Meetup
```
Comment (riacataquian) [inline web/templates/meetup/index.html.slim]: `Alchemists PH` :)

File: web/templates/meetup/index.html.slim [hunk 4]
```diff
@@ -0,0 +1,30 @@
+.navigation-menu
+  .left
+    = link "", to: page_path(@conn, :index), class: "brand-icon"
+  .right
+    .hamburger-menu
```
Comment (riacataquian) [inline web/templates/meetup/index.html.slim]: :yellow_heart:

File: web/templates/meetup/index.html.slim (hunk 2 above)
Comment (samrosecruz) [inline web/templates/meetup/index.html.slim]: lel

File: web/templates/shared/call-for-speakers.html.slim [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.meetup-speakers
+  h2.heading Call for Speakers
+  p.description The floor is open to anyone who has an Elixir story to share. We would love to hear from you! Just a few notes:
+  ul
+    li No previous speaking experience is required, we welcome first timers.
+    li We encourage beginner-friendly talks.
+    li Talks should be 45 minutes long, including Q&A which should only take 5-8 minutes.
+    li Please take note that the meet up will be broadcasted on Google Hangouts.
+  p.description 
+    | Interested in speaking? We would love to hear from you! Feel free to drop your proposal
```
Comment (riacataquian) [inline web/templates/shared/call-for-speakers.html.slim]: Also, @samrosecruz please remove the `We would to hear from you!` phrase. Its kinda redundant with [this](https://github.com/Alchemists-PH/Website/pull/19/files#diff-ae18019d95890226a229ba287d31aaa8R3). I also removed that phrase na sa ticketbase. Sorry for that. :D

File: web/templates/meetup/index.html.slim (hunk 4 above)
Comment (samrosecruz) [inline web/templates/meetup/index.html.slim]: wat

File: web/templates/shared/call-for-speakers.html.slim [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  p.description The floor is open to anyone who has an Elixir story to share. We would love to hear from you! Just a few notes:
+  ul
+    li No previous speaking experience is required, we welcome first timers.
+    li We encourage beginner-friendly talks.
+    li Talks should be 45 minutes long, including Q&A which should only take 5-8 minutes.
+    li Please take note that the meet up will be broadcasted on Google Hangouts.
+  p.description 
+    | Interested in speaking? We would love to hear from you! Feel free to drop your proposal 
+  .action
+    = link 'Submit a Proposal', to: "#", class: "button-ex -primary"
```
Comment (riacataquian) [inline web/templates/shared/call-for-speakers.html.slim]: @samrosecruz can we redirect this to `proposal_path(@conn, :new)` please. :D

File: web/templates/shared/meetups-list.html.slim [hunk 7]
```diff
@@ -0,0 +1,7 @@
+.meetup-list
+  h2.heading Past Meetups
+  ul.list
+    li.item
+      = link "Alchemist |> Meetup 01", to: "", class: "name"
```
Comment (riacataquian) [inline web/templates/shared/meetups-list.html.slim]: `Alchemists PH |> Meetup 01` :)

File: web/static/css/components/meetup-speakers.scss (hunk 1 above)
Comment (samrosecruz) [inline web/static/css/components/meetup-speakers.scss]: wag. hindi ito ung proper. I was trying to target `last-of-type` sana but it wont work </3

File: web/templates/meetup/index.html.slim (hunk 4 above)
Comment (riacataquian) [inline web/templates/meetup/index.html.slim]: why, fuds

File: web/templates/shared/meetups-list.html.slim [hunk 8]
```diff
@@ -0,0 +1,7 @@
+.meetup-list
+  h2.heading Past Meetups
+  ul.list
+    li.item
+      = link "Alchemists |> Meetup 01", to: "", class: "name"
```
Comment (riacataquian) [inline web/templates/shared/meetups-list.html.slim]: `Alchemists PH |> Meetup 01` :)

File: web/templates/meetup/index.html.slim (hunk 4 above)
Comment (samrosecruz) [inline web/templates/meetup/index.html.slim]: image

File: web/templates/shared/call-for-speakers.html.slim [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+  p.description The floor is open to anyone who has an Elixir story to share. We would love to hear from you! Just a few notes:
+  ul
+    li No previous speaking experience is required, we welcome first timers.
+    li We encourage beginner-friendly talks.
+    li Talks should be 45 minutes long, including Q&A which should only take 5-8 minutes.
+    li Please take note that the meet up will be broadcasted on Google Hangouts.
+  p.description 
+    | Interested in speaking? Feel free to drop your proposal 
+  .action
+    = link 'Submit a Proposal', to: "#", class: "button-ex -primary"
```
Comment (riacataquian) [inline web/templates/shared/call-for-speakers.html.slim]: @samrosecruz can we redirect this to `proposal_path(@conn, :new)` please. :)

File: web/static/css/components/meetup-speakers.scss (hunk 1 above)
Comment (riacataquian) [inline web/static/css/components/meetup-speakers.scss]: Still, 💛

**Comment by riacataquian** [comment]
Gandaaaa, thanks @samrosecruz! 💛

**Pull request merged**